fluent-pseudo/Cargo.toml

```diff
@@ -4,7 +4,8 @@ description = """
 Pseudolocalization transformation API for use with Project Fluent API.
 """
 version = "0.3.1"
-edition = "2021"
+edition.workspace = true
+rust-version = "1.64.0"
```

Comment:
Why is this one and another different? (Or more simply, where is this derived from?)

Reply:
Because it *is* different, and I didn't want to artificially boost the MSRV until we decide to do so for some reason, I was only trying to document the status quo. Again this is determined through trial and error with `cargo msrv` as I was feeling my way around where things are at now.

Reply:
This *probably* doesn't matter since the two primary dependents (`fluent` and `l10n`) both have higer MSRV's than this anyway (the latter being incorrectly documented). Of course that doesn't mean *nobody* else directly depends on this without also depending on other parts of Fluent; but chances are I'd be up for normalizing this in the next (non-safe-harbor) release so the whole workspace specifies the MSVR as the highest minimum of all the members.